rstream/client.py

```diff
@@ -256,6 +286,9 @@ async def close(self) -> None:
                 resp_schema=schema.CloseResponse,
             )
 
+        for subscriber_name in self._frames:
+            await self.stop_task("run_delivery_handlers" + subscriber_name)
```

Comment:
Just wanted to notice that we don't wait for all enqueued frames to be handled at shutdown, is it OK? Probably it's fine, since streams protocol has offsets to specify where to start from next time, but I'm not 100% sure